NewsFeed/views/CCBRNewsViewController.m

```diff
@@ -85,6 +87,7 @@ - (void)viewDidLoad {
 - (void)updateUI {
     self.collectionView.hidden = self.viewModel.collectionViewHidden;
     self.errorMessageLabel.hidden = self.viewModel.errorMessageLabelHidden;
+    self.collectionView.hidden = !(self.errorMessageLabel.hidden);
```

Comment:
The logic which decide if collectionView is hidden or not should all be included in ViewModel's collectionViewHidden property. So we have only 1 place to decide collectionView's visibility. It will be easier for read and maintain the code.